api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -231,4 +240,40 @@ public EmailVerificationStatus getEmailVerificationStatus() {
   public void setEmailVerificationStatus(EmailVerificationStatus emailVerificationStatus) {
     this.emailVerificationStatus = emailVerificationStatus;
   }
+
+  @OneToMany(fetch = FetchType.EAGER, mappedBy = "userId", orphanRemoval = true, cascade = CascadeType.ALL)
+  @OrderColumn(name="order_index")
+  public List<InstitutionalAffiliation> getInstitutionalAffiliations() {
+    return institutionalAffiliations;
+  }
+
+  public void setInstitutionalAffiliations(List<InstitutionalAffiliation> newInstitutionalAffiliations) {
+    this.institutionalAffiliations = newInstitutionalAffiliations;
+  }
+
+  public void clearInstitutionalAffiliations() {
```

Comment:
Clients *could* also call getInstitutionalAffiliations().clear(), add(), addAll(), etc. (You don't necessarily have to expose this method or addInstitutionalAffiliation.) I don't mind, just noting.